index.js

```diff
@@ -7,33 +27,221 @@ function ask(questionText) {
   });
 }
 
-// remember the StateMachine lecture
-// https://bootcamp.burlingtoncodeacademy.com/lessons/cs/state-machines
-let states = {
-  'roomOne': { canChangeTo: [ 'roomTwo' ] },
-  'roomTwo': { canChangeTo: [ 'roomThree' ] },
-  'roomThree': { canChangeTo: [ 'roomOne' ] }
+const busStop = new Room();
+  this.roomInv = ["sticks", "bench"]
+busStop.message = () => {
```

Comment:
If you set up your constructor you can do this when you create your object of class `Room` instead of setting it on the object after you've created it.  e.g.`const busStop = new Room(["sticks", "bench"], function(){console.log("your message here...")})